### DISCUSSION TITLE
blog/becoming-an-brilliant-programmer-01/

### DISCUSSION THREAD

**Discussion by giscus** [sqybi/SQYBI.com#22] [Announcements]
# blog/becoming-an-brilliant-programmer-01/

如果需要查看系列中的其它文章，请使用 tag 跳转到文章目录界面：#becoming-an-brilliant-programmer。

https://sqybi.com/blog/becoming-an-brilliant-programmer-01/

**Comment by MD-20880** [comment]
感谢前辈的分享，能够在程序员的路上看到这样的分享十分有意义，期待前辈的更新(*'▽')

**Reply by sqybi** [reply]
不客气！
不过这个系列的文章还是挺难写的，需要足够多的思考时间。前段时间因为在换工作所以闲了很久，才有空去把这些思考整理成文字；但是最近到了新公司又忙起来了，就很难再继续写这个系列了，这也是挺让人烦恼的事。
希望后面工作上慢慢稳定下来之后，能多些空闲去思考和落笔，这个系列真正投入进去写的时候还是很爽的。
